app/views/support_tickets/academy_details.html.erb

```diff
@@ -17,7 +17,9 @@
         Which academy trust do you work for?
       </h1>
 
-      <%= f.govuk_text_field :academy_name, label: {text: 'Academy trust name', size: 's'} %>
+      <%#= f.govuk_text_field :academy_name, label: {text: 'Academy trust name', size: 's'} %>
+      <%= f.govuk_collection_select :academy_name, SupportTicket::AcademyDetailsForm::OPTIONS,
```

Comment:
```suggestion
      <%= f.govuk_collection_select :academy_name, f.multi_academy_trusts,
```

and then add this method to the form object (because an ActiveRecord scope in a constant feels icky):

```ruby
def options
  Trust.gias_status_open.multi_academy_trust.order(:name)
end
```